package.json

```diff
@@ -69,6 +69,7 @@
     "remark-html": "^5.0.1",
     "request": "^2.79.0",
     "sinon": "^1.15.4",
+    "st": "^1.2.0",
```

Comment:
It's [here](https://github.com/mapbox/mapbox-gl-js/blob/af078d0ebd717dc8af5ad47dfdec468d94ab3183/test/integration/package.json#L14-L14). Is that dependency not working for you?